.gitignore

```diff
@@ -108,3 +108,9 @@ src/
 
 # Hide translation .po file copies
 build/translations/*
+
+# Local venv
+.adxvenv/*
+.local/*
+venv
```

Comment:
that's duplicate of line :91 . Could you move all of this new exclusion under the #Enviroments section?